Jenkinsfile

```diff
@@ -0,0 +1,61 @@
+#!groovy
```

Comment:
gaigepr wants to merge 5 commits into master from add-jenkinsfile

Reply:
we should at least do RC and master, if not release

Reply:
done